src/test/java/com/google/cloud/tools/app/CloudSdkAppEngineDeploymentTest.java

```diff
@@ -59,6 +58,9 @@
   public void setUp() throws IOException {
     appYaml1 = tmpDir.newFile("app1.yaml");
     appYaml2 = tmpDir.newFile("app2.yaml");
+
+    sdk.setAppCommandOutputFormat("json");
```

Comment:
I originally added it just to show what consuming it would look like. It doesn't do anything.